crates/sui-framework/sources/governance/staking_pool.move

```diff
@@ -113,21 +91,27 @@ module sui::staking_pool {
 
     /// Create a new, empty staking pool.
     public(friend) fun new(validator_address: address, starting_epoch: u64, ctx: &mut TxContext) : StakingPool {
+        let exchange_rates = table::new(ctx);
+        table::add(
+            &mut exchange_rates,
+            tx_context::epoch(ctx),
+            PoolTokenExchangeRate { sui_amount: 0, pool_token_amount: 0 }
+        );
         StakingPool {
             validator_address,
             starting_epoch,
```

Comment:
Looks like every call site uses `starting_epoch = `tx_context::epoch(ctx) + 1` so we eliminate that param and use `tx_context::epoch(ctx) + 1` directly here